axelor-account/src/main/resources/views/MoveLine.xml

```diff
@@ -596,8 +597,8 @@
         <button name="showInvoiceTermsBtn" title="Show related invoice term(s)"
           onClick="save,action-move-line-view-show-related-invoice-terms"
           readonlyIf="!invoiceTermList.length"/>
-        <field name="invoiceTermList" colSpan="12" form-view="invoice-term-move-line-form"
-          grid-view="invoice-term-move-line-grid"/>
+        <field name="invoiceTermList" colSpan="12" onChange="save"
```

Comment:
we can't save a moveLine each time we update the invoiceTermList, else we will have an account move not correctly balanced on the daybook status.